### PR TITLE
fix(readme): instructions for Laravel 5.5+

### PR DESCRIPTION
**Pull request by emzet**
See #31

### CODE REVIEW

**Review by owenvoke — COMMENTED** [review]
I think this should be `:^1.5.2`?

**Comment by emzet** [comment]
`composer require elhebert/laravel-sri ^1.5.2` (works under Windows CMD)
or `composer require elhebert/laravel-sri:^1.5.2`
but not `composer require elhebert/laravel-sri@^1.5.2`

**Comment by owenvoke** [comment]
Yeah, I was just thinking of sticking with what the Composer docs use in their examples (using colon instead of space as a separator). However, looking at the CLI, you're correct that space, colon, and even equals can all be used as version separators. 🤔

**Comment by emzet** [comment]
So, do you think colon would be better than space?

**Comment by owenvoke** [comment]
I'd say so for consistency, but up to you. 👍

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Troy Hunt wrote an article speaking on the subject, you can read it [here](https
 $ composer require elhebert/laravel-sri
 
 # Laravel 5.5+
-$ composer require elhebert/laravel-sri@^1.5.2
+$ composer require elhebert/laravel-sri ^1.5.2
```
Comment (Elhebert) [inline README.md:25]: Could you use the `:` separator instead?

**Review by Elhebert — CHANGES_REQUESTED** [review]
Thx for the fix.

Small change requested then it's good.

**Comment by Elhebert** [comment]
I created a new branch for the `v1` (Laravel 5.5+).

I updated the README accordingly in https://github.com/Elhebert/laravel-sri/commit/df7af56ba23aacb518b5cbc0289db4acf494cd06

Thx for reporting the issue!

**Pull request closed without merging**